docs/develop/topics/custom_fields.md

```diff
@@ -154,9 +152,9 @@ _("Program") }}: {{value.get("program", "Unknown")}}) {{ ", " if not loop.last
 }} {% endfor %}
 ```
 
-## Applying configuration
+## Configuration
 
-We have defined our field, so now we need to add it to the `RDM_CUSTOM_FIELDS` and `RDM_CUSTOM_FIELDS_UI` configuration:
+Finally, we need to add the new field to the `RDM_CUSTOM_FIELDS` and `RDM_CUSTOM_FIELDS_UI` configuration:
```

Comment:
@zzacharo here I am missing the import. How do I import `ExperimentsCF`?

Reply:
There is no import, the field was implemented above (line 32). So there is the assumption that is on the same file (invenio.cfg) or the user knows where they put it (e.g. their own module). True that we should specify this.